apps/site/components/SlideOver/SlideOverUsage.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  SlideOver,
+  SlideOverHeader,
+  SlideOverProps,
+  useFadeEffect,
+  useUI,
+} from '@faststore/ui'
+
+export type SlideOverUsageProps = {
+  isOpen: SlideOverProps['isOpen']
+  size: SlideOverProps['size']
+  direction: SlideOverProps['direction']
+}
```

Comment:
We can use the [utility type Pick](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) to simplify this type: 

```suggestion
export type SlideOverUsageProps = Pick<SlideOverProps, "isOpen" | "size" | "direction">
```